db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
```

Comment:
Can you manually edit this so that encoding is not changed, but only the fields you wanted to change?

Reply:
I am still investigating why rails tries to change the encoding.